apps/server/src/shared/repo/course/course.repo.ts

```diff
@@ -7,7 +7,7 @@ import { Counted, EntityId } from '@shared/domain/types';
 import { BaseRepo } from '../base.repo';
 import { Scope } from '../scope';
 
-class CourseScope extends Scope<Course> {
+export class CourseScope extends Scope<Course> {
```

Comment:
it would be good if you move this into its own file since you export it now. There should also be unit tests for it.